crates/re_data_source/src/data_loader/loader_rrd.rs

```diff
@@ -51,7 +51,7 @@ impl crate::DataLoader for RrdLoader {
         Ok(())
     }
 
-    fn load_from_file_contents(
+    fn load_from_path_contents(
```

Comment:
`load_from_file_contents` seems more descriptive to me. This won't be used for folders after all, but it could be used for data that didn't come from disk, e.g. the result of a http fetch

Reply:
Hmm yeah that was kinda my point when I renamed this: in the future I see this evolving as `load_from_uri_contents`, and the `uri` might not be from a file at all (e.g. it's just fetching a blob from a MySQL somewhere).

I'll go back to `load_from_file_contents` for now, until we have arbitrary URIs.